integration-tests/src/test/java/oracle/weblogic/kubernetes/ItStickySession.java

```diff
@@ -209,6 +210,18 @@ public void testSameSessionStickinessUsingVoyager() {
     List<String>  hostNames =
         installVoyagerIngressAndVerify(domainUid, domainNamespace, ingressName, clusterNameMsPortMap);
 
+    withStandardRetryPolicy
```

Comment:
I looked at https://jira.oraclecorp.com/jira/browse/OWLS-87825. It's failing while creating ingress, my suggestion was to call create ingress in loop.

Reply:
changed